### PR TITLE
Removed misleading and duplicate debug message

### PR DESCRIPTION
**Pull request by davetcoleman**
This debug message will print when moveit starts up if you have joint states being published from two different locations. The problem is it is not waiting for both messages to be received so it always has this debug message. It is not needed, though, because a proper warning for the same thing is implemented one function call higher, [here](https://github.com/davetcoleman/moveit_ros/blob/73def2db47038d470098412ea45171aa39f68e8c/planning/planning_scene_monitor/src/planning_scene_monitor.cpp#L927) - and that one has a timer to wait for a second at at startup

### CODE REVIEW

**Comment by isucan** [comment]
I understand this is indeed a duplicate message. I do not mind merging this, but if someone uses the CurrentStateMonitor class directly (outside the PlanningSceneMonitor) they will no longer get any message. I'd rather have two messages in some cases than none in some other. Could we maybe reword the warning to make it less confusing?

**Comment by davetcoleman** [comment]
There are 3 `haveCompleteState()` functions and I am only removing the message from the one that returns a vector of missing state strings. Given the user has requested further information about the missing states by using this version of the function, I do not think it needs to also automatically output the debug message.

Additionally, there is no where else in MoveIt! using that version of the function except for the location that already has its own warning messages.

I think it is still appropriate to remove.

**Comment by isucan** [comment]
Agreed :)
